### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by Kesin11**

### CODE REVIEW

File: .github/workflows/linux_test.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: "Linux test"
+on: [push]
```
Comment (azu) [inline .github/workflows/linux_test.yml]: add `pull_request`?

```suggestion
on: [push, pull_request]
```

https://help.github.com/en/articles/workflow-syntax-for-github-actions#on

**Comment by Kesin11** [comment]
Sorry, I created it for the wrong repository.

File: .github/workflows/linux_test.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: "Linux test"
```
Comment (azu) [inline .github/workflows/linux_test.yml]: We want to test on cross platform - windows, linux, macOS.

So, name/file name to be `ci`.

File: .github/workflows/linux_test.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - run: yarn run bootstrap
+
+    # script
+    - run: yarn run test:docs
+    - run: yarn run website
+
+  test:
+    strategy:
+      matrix:
+        node: [8.x, 10.x, 12.x]
```
Comment (azu) [inline .github/workflows/linux_test.yml]: Can you add `os: [macOS-10.14, windows-2016, ubuntu-18.04]` to `matrix:` like this?

https://github.com/textlint/textlint/issues/632#issuecomment-544221780

**Comment by Kesin11** [comment]
Thank you for comment!
I trying in my forked repository https://github.com/Kesin11/textlint/pull/1

I'll try to correct it according to your comments.

**Pull request closed without merging**